code/modules/organs/organ_icon.dm

```diff
@@ -80,23 +80,24 @@ var/global/list/limb_icon_cache = list()
 		mob_icon.Blend(mark_s, ICON_OVERLAY) //So when it's on your body, it has icons
```

Comment:
Not going to do the same up here?

Reply:
Markings don't have a separate mob layer afaik

Reply:
They don't, they're blended into mob_icon right there, but you can see on markings and lips and stuff, it's adding them to overlays on the head, which is for the same purpose (showing up on the severed head)

Reply:
Isn't the mob_icon here just the icon for the limb though? Each called separately from here to update_icons_body() to be assembled into the full body. Like the icon caching builds the mob from pre-painted parts made by this thing. The eyes and lips are applied onto the full mob tho and could be given the same treatment

Reply:
mob_icon is what update_icons_body() applies onto the mob's sprite itself. Anything added to overlays here is totally invisible because this object is inside the body and thus not visible to anyone until it's dropped and becomes it's own atom. So markings, lips, eyes, for their overlays thing, can be deferred until it's dropped.

Same for every limb, really, if you wanted to make this super-complete and do it on all the limbs and update the larger /removed.

Reply:
yeah noticed now